### ISSUE TITLE
add medm desktops launchers

### ISSUE THREAD

**Issue by stefco** [stefco/geco_vm#54]
how to:
http://askubuntu.com/questions/141229/how-to-add-a-shell-script-to-launcher-as-shortcut

**Comment by stefco** [comment]
got dem scripts working finally. made some icons in gimp.

also, here are the favorites in the launcher, in the order I want (for some reason it doesn't seem to work when I try to set them from the install script):

``` bash
$ gsettings get com.canonical.Unity.Launcher favorites
['nautilus-home.desktop', '/home/vagrant/.medm_launcher/medm-lho.desktop', '/home/vagrant/.medm_launcher/medm-llo.desktop', 'gnome-terminal.desktop', 'firefox.desktop', 'libreoffice-writer.desktop', 'libreoffice-calc.desktop', 'libreoffice-impress.desktop']
```

I think we would want to type

``` bash
$ gsettings set com.canonical.Unity.Launcher favorites
['nautilus-home.desktop', '/home/vagrant/.medm_launcher/medm-lho.desktop', '/home/vagrant/.medm_launcher/medm-llo.desktop', 'gnome-terminal.desktop', 'firefox.desktop', 'libreoffice-writer.desktop', 'libreoffice-calc.desktop', 'libreoffice-impress.desktop']
```

wherein the only difference is `set` vs `get`.

**Comment by stefco** [comment]
implemented in f2a6091:

![medm-desktop-launcher-working](https://cloud.githubusercontent.com/assets/9324499/14693173/96e9bd94-072a-11e6-90d8-1e5d59540fa1.png)
